common/views/components/SpacingComponent/SpacingComponent.js

```diff
@@ -4,7 +4,7 @@ import type { Node } from 'react';
 import { classNames } from '../../../utils/classnames';
 
 type Props = {|
-  children: Node,
+  children?: Node,
 |};
```

Comment:
Does make me think a little bit that it should be called a `SpacingShim` - let's see how we go.